src/commands/helm-lint.yaml

```diff
@@ -1,5 +1,25 @@
 ---
+parameters:
+  disable:
+    description: "Disable linting alogether"
+    type: "boolean"
+    default: false
+  lenient:
+    description: "Use more lenient lint rules"
+    type: "boolean"
+    default: false
 description: Lint Helm charts
 steps:
-  - run: |
-      ct lint --validate-maintainers=false --check-version-increment=false --chart-dirs helm
+  - unless:
+      condition: <<parameters.disable>>
```

Comment:
How about actually forcing us to add those dummy values? Would that hurt? Where could we keep them?

Reply:
Dummy values are kept in the code repo with the Helm chart (in `ci` directory`) https://github.com/helm/chart-testing/blob/master/doc/ct_lint.md

> Charts may have multiple custom values files matching the glob pattern '*-values.yaml' in a directory named 'ci' in the root of the chart's directory. The chart is linted for each of these files. If no custom values file is present, the chart is linted with defaults.

So in the long run, I expect people to add dummy values files but I didn't want to enforce that straight away as it would add additional unplanned work for people.

Reply:
Would it mean for example `helm/ignition-operator/ci/installation-values.yaml`? So we don't even need to change the job for that right?